pkgs/development/python-modules/pynut2/default.nix

```
@@ -0,0 +1,36 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, requests
+#, pytestCheckHook
+}:
+
+buildPythonPackage rec {
+  pname = "pynut2";
+  version = "2.1.2";
+
+  src = fetchFromGitHub {
+    owner = "mezz64";
+    repo = "python-nut2";
+    rev = "${version}";
+    sha256 = "1lg7n1frndfgw73s0ssl1h7kc6zxm7fpiwlc6v6d60kxzaj1dphx";
+  };
+
+  propagatedBuildInputs = [
+    requests
+  ];
+
+  # tests are broken, they import 'nut2', not 'pynut2.nut2'
+  #checkInputs = [
+  #  pytestCheckHook
+  #];
+
+  pythonImportsCheck = [ "pynut2.nut2" ];
+
+  meta = with lib; {
+    description = "API overhaul of PyNUT, a Python library to allow communication with NUT (Network UPS Tools) servers.";
+    homepage = "https://github.com/mezz64/python-nut2";
+    license = with licenses; [ gpl3 ];
```

License is [`GPLv3+`](https://github.com/mezz64/python-nut2/blob/master/setup.py#L18).

```suggestion
    license = with licenses; [ gpl3Plus ];
```